source/administration/config-settings.rst

```diff
@@ -2042,9 +2042,23 @@ This filter uses the permissions of the **Bind Username** account to execute the
 | This feature's ``config.json`` setting is ``"UserFilter": ""`` with string input.                                                                                    |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Guest Filter
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+(Optional) Enter an AD/LDAP Filter to use when searching for external users who have Guest Access to Mattermost. Only the objects selected by the query will be accessible to Mattermost.
```

Comment:
The second line isn't correct.  Possibly, add "as Guests." to the end. But other users, can still access Mattermost. Also, should be users, not objects.

Reply:
Changed it to:
"Optional) Enter an AD/LDAP Filter to use when searching for external users who have Guest Access to Mattermost. Only the users selected by the query will be able to log in to and use Mattermost as Guests."